package.json

```diff
@@ -20,13 +19,31 @@
     "webpack-dev-server": "^2.9.3"
   },
   "babel": {
-    "presets": [["env"]],
+    "presets": [
```

Comment:
Why changing the style of these?

Reply:
Probably just auto fixed by the IDE. I'm consider adding prettier to this project sometime in the future.
I'll format this file with compact style first

Reply:
Now fixed.